core/src/main/scala/cats/syntax/traverse.scala

```diff
@@ -2,3 +2,4 @@ package cats
 package syntax
 
 trait TraverseSyntax extends Traverse.ToTraverseOps
+trait TraverseSyntaxBinCompat0 extends UnorderedTraverse.ToUnorderedTraverseOps
```

Comment:
shall we just name this one `UnorderedTraverseSyntax` since we dont' have one?

Reply:
We could. Is there any merit to keeping it as similar to Foldable/UnorderedFoldable (which doesn't have `UnorderedFoldableSyntax`)?

Reply:
`Foldable/UnorderedFoldable` is the unconventional one. I am referring to 
`trait FoldableSyntax extends Foldable.ToFoldableOps with UnorderedFoldable.ToUnorderedFoldableOps`
The convention in Cats is that the syntax trait of a TC does NOT extend the syntax of its parent TC. For users to get the parent syntax, they have to either explicitly import the parent syntax, or using the all inclusive import. 
So I think having an `UnorderdTraverseSyntax` trait is more conventional.

Reply:
sorry about the bike shedding here. The main reason I am discussing more about it is that this would set up a precedence for working around bin compat breakage. although it's a bit atypical